requirements.txt

```diff
@@ -1,2 +1,6 @@
-mock==2.0.0
-termcolor==1.1.0
+coverage
+flake8
+mock
+nose
+termcolor
+tox
```

Comment:
Why is there no package version? Like this we open ourselves to the risk of bug due to package version changes.